fr/django_start_project/README.md

```diff
@@ -112,6 +112,28 @@ ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
 > **Note** : Si vous utilisez un Chromebook, ajoutez cette ligne à la fin de votre fichier settings.py : `MESSAGE_STORAGE = « django.contrib.messages.storage.session.SessionStorage »`
 > 
 > Ajoutez `.amazonaws.com` à `ALLOWED_HOSTS` si vous utilisez cloud9
+> 
+> Si vous hébergez votre projet sur `Glitch.com`, nous devons protéger la clé secrète Django qui doit rester confidentielle (sinon, toute personne remixant votre projet pourrait la voir) : * Tout d'abord, nous allons créer une clé secrète aléatoire. Ouvrez à nouveau le terminal Glitch et tapez la commande suivante : {% filename %}Terminal{% endfilename %} 
```

Comment:
If I remember correctly, `{% filename %}Terminal{% endfilename %}` needs to be on a line of its own for the rendering to look right. Let's update the translation now that #1714 has been merged.

(also for the `{% filename %}` occurances below)